### PR TITLE
chore: Added "strict" value to compiler options in tsconfig

### PR DESCRIPTION
**Pull request by stephenwf**
Alternative to #25 with all strict options. Build fixed by #23, #27 and #28.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "target": "es2019",
     "module": "commonjs",
     "outDir": "build",
-    "alwaysStrict": true,
```
Comment (thewilkybarkid) [inline tsconfig.json:6]: Is this not needed anymore?

Comment (stephenwf) [reply tsconfig.json:6]: Technically not, but could be useful to add the more verbose options for clarity. From the docs:
> Enable all strict type checking options.Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization.

**Comment by thewilkybarkid** [comment]
Going to revisit after ~#64~ #115, as the jsonld library is incompatible.

**Comment by thewilkybarkid** [comment]
Just tried apply to #115 and good news in that it's basically there.

**Comment by thewilkybarkid** [comment]
After https://github.com/libero/article-store/pull/164, the remaining problems are purely related to dependency typings (1 without types, and a few case of using modules that aren't typed).

**Review by thewilkybarkid — APPROVED** [review]
That was quite a lot of work to satisfy a 1-line chore. 😅

**Pull request merged**